llvm/lib/Target/AArch64/AArch64Processors.td

```diff
@@ -447,6 +447,16 @@ def TuneNeoverseN2 : SubtargetFeature<"neoversen2", "ARMProcFamily", "NeoverseN2
                                       FeatureEnableSelectOptimize,
                                       FeaturePredictableSelectIsExpensive]>;
 
+def TuneNeoverseN3 : SubtargetFeature<"neoversen3", "ARMProcFamily", "NeoverseN3",
+                                      "Neoverse N3 ARM processors", [
+                                      FeatureFuseAES,
+                                      FeaturePostRAScheduler,
+                                      FeatureCmpBccFusion,
```

Comment:
Hi - Should FeatureCmpBccFusion be enabled over what is in N2? The core might well be able to fuse them, but I don't think that is new and from what I remember the performance was sometimes worse with it enabled. (I think llvm's implementation of FeatureCmpBccFusion might be a bit more aggressive than is helpful).

Reply:
Thanks for the suggestion, Dave. I've removed the `FeatureCmpBccFusion` feature flag in a new commit, as we don't want performance to suffer.